app/components/Transfers/configureDelegation/DelegationTargetPage.tsx

```diff
@@ -128,24 +128,32 @@ export default function DelegationTargetPage({
                             }}
                         />
                         <p className="mB30">
-                            If you don&apos;t already know what baker pool you
-                            want to delegate an amount to, you can read more
-                            about finding one here:
+                            If you don&apos;t already know which baker pool you
+                            want to delegate an amount to, you can look for one
+                            here:
+                        </p>
+                        <p className="mV0">
+                            <ExternalLink href={urls.ccdScanStakingPage}>
+                                CCDScan.io/staking
+                            </ExternalLink>
                         </p>
```

Comment:
I guess it could link to the testnet one for the testnet wallet, but that is probably overkill for this.

Reply:
No I think we should do that, we just didn't do that before.

Reply:
But note that we just use the testnet link for everything not mainnet. We don't have ccdscan for stagenet as far as I know.

Reply:
We have it at:
https://stagenet.ccdscan.io/staking :shrug: